### PR TITLE
Travel recommendation added

### LINKED ISSUE #68: add support for Travel Recommendations API
https://developers.amadeus.com/self-service/category/trip/api-doc/travel-recommendations/api-reference

### PR DESCRIPTION
**Pull request by minjikarin**
fixed #68 
added Travel recommendation API

### CODE REVIEW

File: src/resources/Location.php [hunk 1]
```diff
@@ -11,6 +11,8 @@
  * @see Locations::get()
  * @see Airports::get()
  * @see https://developers.amadeus.com/self-service/category/air/api-doc/airport-and-city-search/api-reference
+ * @see RecommendedLocations::get()
+ * @see https://developers.amadeus.com/self-service/category/trip/api-doc/travel-recommendations/api-reference
```
Comment (tsolakoua) [inline src/resources/Location.php:15]: Add the docs of Airport Nearest Relevant API as well, and on line 10 Travel Recommendations API

**Comment by minjikarin** [comment]
@tsolakoua  - readdddy!

**Comment by tsolakoua** [comment]
@minjikarin  update in README is missing in the list of supported endpoints :)

File: tests/referenceData/RecommendedLocationsTest.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<?php
```
Comment (tsolakoua) [inline tests/referenceData/RecommendedLocationsTest.php:1]: @xianqiliu should we add the KO file as well or not? Let's make a decision so we can be consistent :)

File: src/resources/Location.php [hunk 3]
```diff
@@ -7,10 +7,13 @@
 use Amadeus\ReferenceData\Locations\Airports;
 
 /**
- * A Location object as returned by the Airport and City Search API, Airport Nearest Relevant API.
+ * A Location object as returned by the Airport and City Search API, Airport Nearest Relevant API and Travel Recommendations API .
  * @see Locations::get()
+ * @see https://developers.amadeus.com/self-service/category/air/api-doc/airport-nearest-relevant
```
Comment (xianqiliu) [inline src/resources/Location.php:12]: @minjikarin my bad! Line 12 and  Line 14 need to be swapped

File: tests/referenceData/RecommendedLocationsTest.php (hunk 2 above)
Comment (xianqiliu) [reply tests/referenceData/RecommendedLocationsTest.php:1]: yes, we should make a decision, next week I will schedule a meeting with the team to review this question together :) @tsolakoua  @minjikarin @jabrena. As for this PR, @minjikarin please add the KO test for making sure the whole team understands how to code the KO test for API.

**Comment by xianqiliu** [comment]
This PR looks fine, thx @minjikarin :)

File: tests/referenceData/RecommendedLocationsTest.php (hunk 2 above)
Comment (jabrena) [reply tests/referenceData/RecommendedLocationsTest.php:1]: Yes, we need to sit down to review the testing process. 🤩

**Pull request merged**